test/helpers/data/staging-court-reports-helper.js

```diff
@@ -0,0 +1,38 @@
+const knex = require('../../../knex').stagingSchema
+var Promise = require('bluebird').Promise
+
```

Comment:
Actually inserting `court_reporters` data. Maybe rename file, or give this function a more descriptive name - a `court-reports-helper` with an `insertCourtReporters` function

Reply:
Change the name, but think the insert dependencies function name is Ok, seem to be what we have in other places